backend/src/stwo/prover.rs

```diff
@@ -165,7 +173,7 @@ where
                                 > = fixed_columns
                                     .iter()
                                     .map(|(_, vec)| {
-                                        gen_stwo_circle_column::<F, B, M31>(
+                                        gen_stwo_circle_column::<B, M31>(
```

Comment:
```suggestion
                                        gen_stwo_circle_column::<_, M31>(
```

`B` is correctly inferred here, and eventually we shouldn't need the other type annotation either, but that can be done separately